corehq/apps/app_manager/templates/app_manager/v1/summary.html

```diff
@@ -36,6 +36,7 @@
                 $httpProvider.defaults.headers.common['X-Requested-With'] = 'XMLHttpRequest';
                 $httpProvider.defaults.xsrfCookieName = 'csrftoken';
                 $httpProvider.defaults.xsrfHeaderName = 'X-CSRFToken';
+                $httpProvider.defaults.headers.common["X-CSRFToken"] = '{{ csrf_token }}';
```

Comment:
Updated the branch with master and ran `build_app_manager_v2_diffs` again. It generated a diff file `corehq/apps/app_manager/tests/data/v2_diffs/templates/summary.html.diff.txt`.